index.js

```diff
@@ -47,7 +37,7 @@ function jaegerPlugin(fastify, opts, next) {
     fastify.decorateRequest('jaeger', api);
   }
 
-  if (!opts.disable) {
+  if (!('JAEGER_DISABLED' in process.env) || ('JAEGER_DISABLED' in process.env && !process.env.JAEGER_DISABLED)) {
```

Comment:
What if JAEGER_DISABLED is overridden by options passed to plugin?
Maybe tracer instance itself has some option to check whether its disabled or not?
If not then you should add check for passed parameter too. Also, see how are we parsing boolean envs in silverfin adaptor (config.ts#getBooleanEnv) - it seems a bit more readable then the above one.

Reply:
Options passed in config object will override env vars. I will implement logic do check if Jaeger is enabled/disabled checking env var or config object property depending on what configuration option was defined.